services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -42,10 +48,11 @@ export const AddEditReportModal = ({
   const [submitting, setSubmitting] = useState<boolean>(false);
 
   const naaarReport = useFlags()?.naaarReport;
+  const mcparProgramList = useFlags()?.mcparProgramList;
```

Comment:
seeing an issue when I toggle this to false

https://github.com/user-attachments/assets/42b806b2-42cc-4d7b-b40b-8b59c7bece7e